downstream/modules/hub/proc-configuring-the-client-to-verify-signatures.adoc

```diff
@@ -66,8 +72,10 @@ name of your key file.
 * Pull the file using Podman, or your client of choice:
 
 ----
-> podman pull _<server-address>_/<container-name>:<tag name>
---tls-verify=false
+> podman pull <server-address>/<container-name>:<tag name> --tls-verify=false
 ----
 
-This response verifies the signature with no errors.
+This response verifies the image has been signed with no errors. If the image is not signed, the command will fail.
```

Comment:
Remove will to avoid use of future tense?

"If the image is not signed, the command fails".